### PR TITLE
Add ECS logger

### PR DESCRIPTION
**Pull request by ruflin**
WIP

### CODE REVIEW

**Comment by ruflin** [comment]
@simitt I started to play around with the zap ECS logger (see changes in PR). Would be great if you could have a quick look if I use it as expected?

The logs currently look as following:

```
{"log.level":"error","@timestamp":"2020-04-24T21:07:21.307+0200","log.origin":{"file.name":"package-registry/main.go","file.line":92},"message":"Error serving: http: Server closed","ecs.version":"1.5.0"}
```

In my case, I don't need the `log.origin` part, is there an option to disable this?

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,9 @@ require (
 	github.com/gorilla/mux v1.7.2
 	github.com/jstemmer/go-junit-report v0.9.1 // indirect
 	github.com/magefile/mage v1.9.0
-	github.com/pkg/errors v0.8.1
-	github.com/stretchr/testify v1.4.0
+	github.com/pkg/errors v0.9.1
```
Comment (ruflin) [inline go.mod:11]: Was not planned to update these as part of this PR, need to revert.

**Comment by ruflin** [comment]
Closing for now as not top priority. Created an issue to keep track of it: https://github.com/elastic/package-registry/issues/418

**Comment by simitt** [comment]
@ruflin you can create the logger without the `zap.AddCaller` option to avoid having the `log.origin` logged; like this `logger = zap.New(core)`

**Pull request closed without merging**